packages/ansible/packaging

```diff
@@ -1,5 +1,107 @@
-set -e
+#!/usr/bin/env bash
 
-ANSIBLE_VERSION=v2.5.2
+# abort script on any command that exits with a non zero value
+set -e -x
+
+# Grab the latest versions that are in the directory
+ANSIBLE_VERSION=$(ls -r ansible/ansible-*.tar.gz | sed 's/.*\/ansible-\(.*\)\.tar\.gz$/\1/' | head -1)
+PYCRYPTO_VERSION=$(ls -r ansible/pycrypto-*.tar.gz | sed 's/.*\/pycrypto-\(.*\)\.tar\.gz$/\1/' | head -1)
+JINJA_VERSION=$(ls -r ansible/jinja-*.tar.gz | sed 's/.*\/jinja-\(.*\)\.tar\.gz$/\1/' | head -1)
+PARAMIKO_VERSION=$(ls -r ansible/paramiko-*.tar.gz | sed 's/.*\/paramiko-\(.*\)\.tar\.gz$/\1/' | head -1)
+LIBFFI_VERSION="3.2.1"
```

Comment:
Does the $(ls -r BLAH-*.tar.gz | sed 's/.*\/BLAH-\(.*\)\.tar\.gz$/\1/' | head -1) command work for LIBFFI?

Reply:
It's a good question. I don't know. Perhaps something to look at at a later date?